lib/meta_inspector/parsers/images.rb

```diff
@@ -10,12 +12,44 @@ def images
         self
       end
 
+      # Returns either the Facebook Open Graph image, twitter suggested image or
+      # the first image in the image collection
+      def best
+        owner_suggested || images_collection.first
+      end
+
```

Comment:
Now that we're able to return the largest image, I'd change #best so that it returns it, instead of the first one, it there is no #owner_suggested image:

```ruby
owner_suggested || largest
```

As I said on the comment on the README, returning the first image was our naïve attempt at returning the best image, now we can improve the results of this method.